extensions/eclipse-che-theia-plugin-ext/src/browser/oauth-utils.ts

```diff
@@ -35,7 +35,7 @@ export class OauthUtils {
             }
         });
         window.addEventListener('message', data => {
-            if (data.data.startsWith('token:') && this.oAuthPopup) {
+            if (this.oAuthPopup && data.data && (typeof data.data === 'string') && data.data.startsWith('token:')) {
```

Comment:
`data.data` check is redundant if the check for `typeof` is present

Reply:
fixed